sdk/linux_kernel_drivers/xdma/libxdma.c

```diff
@@ -62,13 +64,38 @@ static unsigned int interrupt_mode;
 module_param(interrupt_mode, uint, 0644);
 MODULE_PARM_DESC(interrupt_mode, "0 - MSI-x , 1 - MSI, 2 - Legacy");
 
-static unsigned int enable_credit_mp;
+static unsigned int enable_credit_mp = 1;
 module_param(enable_credit_mp, uint, 0644);
-MODULE_PARM_DESC(enable_credit_mp, "Set 1 to enable creidt feature, default is 0 (no credit control)");
+MODULE_PARM_DESC(
+	enable_credit_mp,
+	"Set 0 to disable credit feature, default is 1 ( credit control enabled)");
```

Comment:
Karen - how changing default impact F1 customers?   sounds like a change that should not be done transparently, and we need to let customers opt-in

Reply:
I will change the default to 1 as in the current xdma driver.

Reply:
Sorry, I mean change the default back to 0.

Reply:
Change is in place. Resolving this conversation.